games-action/minetest/minetest-0.4.12.ebuild

```diff
@@ -0,0 +1,157 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+
+EAPI=5
+inherit eutils cmake-utils gnome2-utils vcs-snapshot user games
+
+DESCRIPTION="An InfiniMiner/Minecraft inspired game"
+HOMEPAGE="http://minetest.net/"
+SRC_URI="http://github.com/minetest/minetest/tarball/${PV} -> ${P}.tar.gz"
+
+LICENSE="LGPL-2.1+ CC-BY-SA-3.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="+curl dedicated doc leveldb luajit nls redis +server +sound +truetype"
+
+RDEPEND="dev-db/sqlite:3
```

Comment:
If you have some time, please take a look to which libraries the game links to, and add `:=` slot operators to appropriate deps.